Base/Python/slicer/parameterNodeWrapper/serializers.py

```diff
@@ -3,16 +3,12 @@
 import logging
 import pathlib
 import typing
+import enum
 
 import slicer
 
 from .util import splitAnnotations
-from .validators import (
-    extractValidators,
-    IsNone,
-    NotNone,
-    IsInstance,
-)
+from .validators import (extractValidators, IsNone, NotNone, IsInstance, Choice, )
```

Comment:
Why was this flattened to one line?

Reply:
PyCharm "helpfully" formatted things before committing. I'll revert this soon.